lib/docblockr-worker.js

```diff
@@ -61,6 +62,8 @@ module.exports =
         var regex = {
           // Parse Command
           'parse': /^\s*(\/\*([*!])|###\*|\/\/\*)\s*$/,
+          // Parse python docstring
+          'parse_py': /^\s*"""\s*$/,
```

Comment:
I think it's time to put the regex for the comment start in the language files and handle this in a language specific way. The `parse` regex already contains check for way to many language specific comment starts (`/**` for c style, `###*` for coffee script, `//*` for scss, `/*!` for what not).